routing_common/transit_serdes.hpp

```diff
@@ -178,6 +183,11 @@ class Deserializer
     id = FeatureIdentifiers(kInvalidOsmId, featureId);
   }
 
+  void operator()(StopIdRanges & rs, char const * name = nullptr)
```

Comment:
Why do we need this? StopIdRanges contains Visit() method, and there is a template operator() below that calls Visit methods explicitly.

Reply:
Right! Let's remove these two operator(StopIdRanges ... ) at this file.